packages/discovery-react-components/src/components/CIDocument/utils/__tests__/details.spec.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  getDetailsFromItem,
+  getDetailsFromMetadata,
+  contractDetailsFromItem,
+  nonContractDetailsFromItem,
+  getAttributesWithinRelation
+} from '../details';
+
+const mockData = {
+  metadataType: 'effective_dates',
+  text: '24th October 2011'
+};
+
+const mockDataNoMeta = { text: '24th October 2011' };
+const mockDetails = [{ heading: 'type', items: ['effective_dates'] }];
+const mockDetailsNoMeta = [{ heading: 'party', items: ['24TH OCTOBER 2011'] }];
```

Comment:
what's the difference between `mockDetails` and `mockDetailsNoMeta`? seems to be the same to me beside the value in it

Reply:
The difference between the two are test the different outputs from the function `getDetailsFromMetadata`.  The heading and items values will change based on if a metadataType is present in the data prop passed in.

Reply:
Will change the variable names to be more clear.